api/net/Areas/Editor/Controllers/StorageController.cs

```diff
@@ -67,11 +66,11 @@ public StorageController(IConnectionHelper connection, IOptions<StorageOptions>
     public IActionResult GetFolder([FromRoute] int? locationId, [FromQuery] string? path)
     {
         path = String.IsNullOrWhiteSpace(path) ? "" : HttpUtility.UrlDecode(path).MakeRelativePath();
-        if (locationId != null)
+        var connection = locationId.HasValue ? _connection.GetConnection(locationId.Value) : null;
```

Comment:
Presently the editor app passes an invalid connection.  The default behaviour should currently be accessing files in the API.  I suspect this is not what we want in the future however.